pyuvdata/uvdata/ms.py

```diff
@@ -119,6 +119,63 @@ def _write_ms_antenna(self, filepath):
             antenna_table.putcol("DISH_DIAMETER", ant_diam_table)
         antenna_table.done()
 
+    def _write_ms_feed(self, filepath):
+        """
+        Write out the feed information into a CASA table.
+
+        Parameters
+        ----------
+        filepath : str
+            path to MS (without FEED suffix)
+
+        """
+        if not casa_present:  # pragma: no cover
+            raise ImportError(no_casa_message) from casa_error
+
+        feed_table = tables.table(filepath + "::FEED", ack=False, readonly=False)
+
+        nfeeds_table = np.max(self.antenna_numbers) + 1
+        feed_table.addrows(nfeeds_table)
+
+        antenna_id_table = np.arange(nfeeds_table, dtype=np.int32)
+        feed_table.putcol("ANTENNA_ID", antenna_id_table)
+
+        beam_id_table = np.ones(nfeeds_table, dtype=np.int32)
```

Comment:
since we're not filling out a beam table, I think this should be filled with -1s. From the doc:
> BEAM ID Beam identifier. Points to an optional BEAM sub-table defining the primary beam and polarization response for this FEED. A value of -1 indicates that no associated beam response is defined.